app/models/metric.rb

```diff
@@ -26,7 +26,9 @@ class Metric < ApplicationRecord
 
   acts_as_tree
 
-  attr_protected :service_id, :parent_id, :tenant_id, :audit_ids
```

Comment:
There must be a reason why `service_id` is protected, you can't just unprotect it.

Reply:
@josemigallas yes agreed and we keep id's as protected attributes only but warning also raise for the same.